test/JDBC/input/views/sys-events-vu-prepare.sql

```diff
@@ -0,0 +1,61 @@
+USE master
+GO
+
+CREATE TABLE sys_events_vu_prepare_table1(c1 int)
+GO
+
+CREATE SCHEMA sys_events_vu_prepare_schema1
+GO
+
+CREATE TABLE sys_events_vu_prepare_schema1.sys_events_vu_prepare_table1(c1 int)
+GO
+
+--create trigger on default schema for master database
+CREATE TRIGGER sys_events_vu_prepare_trig1 ON sys_events_vu_prepare_table1 INSTEAD OF DELETE
+AS
+BEGIN
+  SELECT 'trigger invoked'
+END
+GO
+
+--create trigger on different schema
+CREATE TRIGGER sys_events_vu_prepare_trig2 ON sys_events_vu_prepare_schema1.sys_events_vu_prepare_table1 FOR INSERT, UPDATE
+AS
+BEGIN
+  SELECT 'trigger invoked'
+END
+GO
+
+CREATE VIEW sys_events_vu_prepare_view AS
+SELECT type, type_desc, is_trigger_event, event_group_type, event_group_type_desc FROM sys.events ORDER BY type
+GO
+
+CREATE PROC sys_events_vu_prepare_proc AS
+SELECT type, type_desc, is_trigger_event, event_group_type, event_group_type_desc FROM sys.events ORDER BY type
+GO
+
+CREATE FUNCTION dbo.sys_events_vu_prepare_func()
+RETURNS INT
+AS
+BEGIN
+RETURN (SELECT COUNT(*) FROM sys.events)
+END
+GO
+
+
+CREATE DATABASE sys_events_vu_prepare_database1
+GO
+USE sys_events_vu_prepare_database1
+GO
+
+CREATE TABLE sys_events_vu_prepare_table1(c1 int)
+GO
+
+--create trigger on default schema for db1 database
+CREATE TRIGGER sys_events_vu_prepare_trig3 ON sys_events_vu_prepare_table1 INSTEAD OF UPDATE
+AS
+BEGIN
+  SELECT 'trigger invoked'
+END
+GO
+
```

Comment:
Thanks for your comment. I can remove creating a new db, but just wanted to inform the purpose of creating a db here is to make sure any procedure objects created in this DB are not present when we switch context to master db. This is the [line of code that tests this](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/454/files/d8cfcbeb4d74eebc7faf4524aa8bd0df79166af5#diff-61981a6eff1c4073f25131037c2b327766867f54c79b28f8d756effd86285ee9R24). Should I still go ahead and remove this?

Reply:
I am not saying to remove these tests. We can add those tests if we require them. Only thing that we need to ensure that we switch back to master before end of prepare script. Otherwise normal JDBC tests will run fine. But any upgrade/prepare script being run after this prepare script will be ran into new database.

Reply:
Done, switched context back to master DB at end of script.